apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -17,13 +146,15 @@
   }
 }
 
-.reactions-container {
-  background-color: #f2f2f2;
-  padding: 20px;
+.webcam-container {
+  background-color: white;
+  margin-bottom: 10px;
 }
 
 .webcam {
-  width: 200px;
+  width: 75%;
+  margin: 0 auto;
```

Comment:
Properties should be ordered width, padding, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)